R/Quandlapi.R

```diff
@@ -1,63 +1,82 @@
 #' Pulls Data from the Quandl API
 #'
-#' An authentication token is needed for access to the Quandl API multiple times. Set your \code{access_token} with \code{Quandl.auth} function.
+#' An authentication token is needed for access to the Quandl API multiple times. Set your \code{access_token} with \code{Quandl.api_key} function.
 #'
-#' For instructions on finding your authentication token go to https://www.quandl.com/account
-#' @param version Set to the version of the Quandl API you want to access.
+#' For instructions on finding your authentication token go to https://www.quandl.com/account/api
 #' @param path Path to api resource.
 #' @param http Type of http request sent.
 #' @param postdata A character or raw vector that is sent in a body.
 #' @param ... Named values that are interpretted as api parameters.
 #' @return Website response.
-#' @references This R package uses the Quandl API. For more information go to https://www.quandl.com/help/api. For more help on the package itself go to http://www.quandl.com/help/r.
+#' @references This R package uses the Quandl API. For more information go to https://www.quandl.com/docs/api. For more help on the package itself go to http://www.quandl.com/help/r.
 #' @author Raymond McTaggart
-#' @seealso \code{\link{Quandl.auth}}
+#' @seealso \code{\link{Quandl.api_key}}
 #' @examples \dontrun{
-#' quandldata = quandl.api(version="v1", path="datasets/NSE/OIL", http="GET")
+#' quandldata = quandl.api(path="datasets/NSE/OIL", http="GET")
 #' plot(quandldata[,1])
 #' }
 #' @export
-quandl.api <- function(version="v1", path, http = c('GET', 'PUT', 'POST', 'DELETE'), postdata = NULL, ...) {
+quandl.api <- function(path, http = c('GET', 'PUT', 'POST', 'DELETE'), postdata = NULL, ...) {
+  http <- match.arg(http)
+  request <- quandl.api.build_request(path, ...)
+  response <- httr::VERB(http, request$request_url, config = do.call(httr::add_headers, request$headers),
+                                      body = postdata, query = request$params)
+
+  quandl.api.handl_errors(response)
+  text_response <- httr::content(response, as="text")
```

Comment:
Please, could you also use everywhere where it is needed: 
```
#' @import httr
#' @import jsonlite
```
E.g. like here https://github.com/quandl/quandl-r/pull/36/files#diff-185842ab024dc31efeaf6b40e8455b34R56